### PR TITLE
Disable pager when loading structure for PostgreSQL

### PR DESCRIPTION
**Pull request by jcoleman**
### Summary

tl;dr In PostgreSQL 9.6.8+ (and similarly patched versions of 10 as well as 11), the `rake db:test:load_structure` task can cause a blocking interactive shell command.

The [fix to CVE-2018-1058](https://wiki.postgresql.org/wiki/A_Guide_to_CVE-2018-1058:_Protect_Your_Search_Path) (among other things) changes pg_dump to emit:
   SELECT pg_catalog.set_config('search_path', '', false);
into the structure.sql file. In certain environments (e.g., docker) this
triggers the pager when psql is used to restore the database structure.
Since the pager results in the rake task containing a blocking
interactive shell command, we need to make sure psql never uses a pager
when running rake tasks for PostgreSQL.

### CODE REVIEW

**Comment by jcoleman** [comment]
The build which failed is GEM=railties in test:
```
ApplicationTests::AssetDebuggingTest#test_assets_are_concatenated_when_debug_is_off_and_compile_is_off_either_if_debug_assets_param_is_provided:
RuntimeError: rails command failed (1): bin/rails assets:precompile --trace 2>&1
```
with the error:
```
ModuleNotFoundError: Module not found: Error: Can't resolve '@rails/activestorage' in '/home/travis/build/rails/rails/tmp/d20190211-23128-1urld1w/app/app/javascript/packs'
```

This appears to be entirely unrelated to my change. Is there an issue on master?

**Comment by tonydehnke** [comment]
Ran into this on my CircleCI testing failing with Postgres and @swrobel was kind enough to show this as the fix.   Here is the relevant thread incase it helps anyone else:   https://discuss.circleci.com/t/postgres-failing-with-upgrading-from-11-6-to-11-14/42932/2

**Comment by jcoleman** [comment]
It's really unfortunate that PRs like this get closed not because of an unresponsive contributor but because no maintainer has even taken the time to comment on them.

**Comment by nightpool** [comment]
Just ran into this today as part of the new Circle upgrade, would be great if this got merged. But unfortunately it seems like yet another opaque workaround everybody is going to have to add to their circle config. Is there anything Circle can do to fix this problem? I'm pretty sure I only changed my base image version, nothing about my PG version should have changed.
